### PR TITLE
add more metrics to gather

### PR DESCRIPTION
**Pull request by deads2k**
Follow on to https://github.com/openshift/insights-operator/pull/40 that adds metrics interesting to group-b.

### CODE REVIEW

File: pkg/gather/clusterconfig/clusterconfig.go [hunk 1]
```diff
@@ -63,8 +63,24 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 			}
 			data, err := i.metricsClient.Get().AbsPath("federate").
 				Param("match[]", "ALERTS").
+				// aggregator metrics indicate how apiservers are performing as viewed from the kube-apiserver
+				Param("match[]", "aggregator_unavailable_apiservice_count").
+				Param("match[]", "aggregator_unavailable_apiserver_gauge").
```
Comment (smarterclayton) [inline pkg/gather/clusterconfig/clusterconfig.go]: do these have alerts?

File: pkg/gather/clusterconfig/clusterconfig.go [hunk 2]
```diff
@@ -63,8 +63,24 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
...
+
+				// let's us know how many of each kind of object we have.  This give information about usage
 				Param("match[]", "etcd_object_counts").
+
 				Param("match[]", "cluster_installer").
+
+				// these work queue metrics can be used to determine how individual control loops are performing
+				Param("match[]", "workqueue_adds_total").
```
Comment (smarterclayton) [inline pkg/gather/clusterconfig/clusterconfig.go]: what's the cardinality of these?  if it's big I want to start with the ones that indicate problems, vs debugging them all.

File: pkg/gather/clusterconfig/clusterconfig.go (hunk 1 above)
Comment (deads2k) [reply pkg/gather/clusterconfig/clusterconfig.go]: > do these have alerts?

Not sure actually.  Not in the kas-o where I would expect them to be defined.

File: pkg/gather/clusterconfig/clusterconfig.go (hunk 2 above)
Comment (deads2k) [reply pkg/gather/clusterconfig/clusterconfig.go]: > what's the cardinality of these? if it's big I want to start with the ones that indicate problems, vs debugging them all.

potentially one per control loop.  Do we make it easy to say: all these metrics from the kas, kcm, ks, oas, and ocm?  Those have the most value, but it also picks up most of the cardinality.

**Pull request closed without merging**